packages/flutter/lib/src/widgets/scroll_metrics.dart

```diff
@@ -36,7 +36,7 @@ abstract class ScrollMetrics {
   ///
   /// This is useful if this object is mutable, but you want to get a snapshot
   /// of the current state.
-  ScrollMetrics cloneMetrics() => new FixedScrollMetrics.clone(this);
+  ScrollMetrics toMetrics() => new FixedScrollMetrics.clone(this);
```

Comment:
https://github.com/dart-lang/linter/blob/master/lib/src/rules/use_to_and_as_if_applicable.dart#L76

Basically method with no parameter that returns a new instance of the same type with a constructor that takes `this` as parameter.